sentences/de/lock_HassTurnOn.yaml

```diff
@@ -0,0 +1,11 @@
+language: de
+intents:
+  HassTurnOn:
+    data:
+      - sentences:
+          - "<sperren> <name> [<area>] [zu|ab]"
+          - "<name> [<area>] zu[<sperren>|machen]"
+          - "<name> [<area>] ab(<sperren>)"
```

Comment:
Sure? Then we're gonna have some grammatically incorrect constellations possible or isn't this a problem?

Reply:
It's really tricky as we definitely need <sperren> without "zu" or "ab" sometimes and these shouldn't be optional in the expansion rule to avoid wrong behavior with locks so I created <absperren> separately.